### PR TITLE
Add firefox/browsers/compare/opera.ftl

### PR DESCRIPTION
**Pull request by craigcook**
[Template](https://github.com/mozilla/bedrock/pull/8958/files#diff-b699ffdd322e7132eaabcaf9321febe5)
[Demo](https://www-demo1.allizom.org/firefox/browsers/compare/opera/)

### CODE REVIEW

File: en/firefox/browsers/compare/opera.ftl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# HTML page title
+compare-opera-firefox-vs-opera-which-is = { -brand-name-firefox } vs. { -brand-name-opera }: Which is the better browser for you?
+
+# HTML page description
+compare-opera-compare-opera-to-the-firefox = Compare { -brand-name-opera } to the { -brand-name-firefox-browser } to find out which is the best browser for you.
+
+# Main title
+compare-opera-comparing-firefox-browser = Comparing { -brand-name-firefox-browser } with { -brand-name-opera }
+
+compare-opera-operas-privacy-policy-lacks = Opera’s privacy policy lacks some specificity in its explanation of which types of information it collects and how, in certain sections, it says they collect names of account holders, IP addresses and search terms. What seems confusing and troubling is the section about International data transfers; when, how often and why they need to transfer your data internationally is not explained.
```
Comment (peiying2) [inline en/firefox/browsers/compare/opera.ftl:16]: 1). `Opera`
2). Change comma to a period followed by a capital I so this long sentence breaks into two: `... it collects and how. In certain sections...`

File: en/firefox/browsers/compare/opera.ftl [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+compare-opera-the-firefox-browser-and-opera = The { -brand-name-firefox-browser } and { -brand-name-opera } are two of the earliest browsers on the scene still releasing frequent updates. While { -brand-name-opera } has not reached the same level of user adoption as { -brand-name-firefox } or { -brand-name-google } { -brand-name-chrome }, it’s maintained a relatively stable and loyal base over a sustained period of time. In this review, we’ll compare the { -brand-name-opera } browser with our { -brand-name-firefox } browser in terms of security and privacy, utility, and portability to help you choose which browser might be the best fit for you.
+
+# Variables:
+#   $attrs (string) - link to /privacy/firefox/ with additional attributes for analytics
+compare-opera-firefoxs-privacy-policy-is = Firefox’s <a { $attrs }>privacy policy</a> is very transparent in describing what personal information we collect with the only end goal being to give you greater control over the information you share online.
```
Comment (peiying2) [inline en/firefox/browsers/compare/opera.ftl:21]: `Firefox`

File: en/firefox/browsers/compare/opera.ftl [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+compare-opera-with-the-latest-version-of = With the latest version of { -brand-name-firefox }, <a { $attrs }>Enhanced Tracking Protection</a> is turned on by default in normal browsing mode, so you don’t have to mess around with the settings just to protect yourself from trackers. With Enhanced Tracking Protection, { -brand-name-firefox } actively blocks thousands of third-party trackers that try to follow you around the web. You are provided with a personalized protection report that shows how often { -brand-name-firefox } blocked third-party cookies, social media trackers, fingerprinting tools and cryptominers as you browse the web.
+compare-opera-we-make-firefox-for-people = We make { -brand-name-firefox } for people like you, who care deeply about personal privacy and security. That's why we collect so little info about users and are transparent about how we use that info. It's hard to know how Opera is operating from a privacy perspective. While there are robust privacy features, how they themselves collect and share your data is murky. { -brand-name-firefox } remains consistent in what we say and what we do in protecting your privacy.
```
Comment (peiying2) [inline en/firefox/browsers/compare/opera.ftl:27]: `Opera`

File: en/firefox/browsers/compare/opera.ftl [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+# "Ex" is a former partner or spouse, as in "ex-boyfriend" or "ex-wife." Alternative: "former partner"
+compare-opera-both-firefox-and-opera-are = Both { -brand-name-firefox } and { -brand-name-opera } are compatible across every platform including { -brand-name-windows }, Mac, { -brand-name-linux }, { -brand-name-android } and { -brand-name-ios }. { -brand-name-firefox } account holders can easily sync their bookmarks, passwords, open tabs, and browsing history across all their signed into devices. The same is true for { -brand-name-opera } users with an account. However, many sites, especially old sites that haven’t been updated in years, block the latest version of { -brand-name-opera } entirely. So if visiting places like your Ex’s old blog is important, take heed, you may not be able to access some of the dustier corners of the internet if you use { -brand-name-opera }.
```
Comment (peiying2) [inline en/firefox/browsers/compare/opera.ftl:41]: `Mac`: shall we change to `macOS` to standardize the reference? Or we can add `Mac` to the brands.ftl.

File: en/firefox/browsers/compare/opera.ftl (hunk 2 above)
Comment (peiying2) [reply en/firefox/browsers/compare/opera.ftl:22]: This one is still there.

Comment (craigcook) [reply en/firefox/browsers/compare/opera.ftl]: oops, fixed now!

**Pull request merged**